spec/features/unlinked_defendant_flow_spec.rb

```diff
@@ -84,8 +85,8 @@ def then_case_view_displayed
     expect(page).to have_css('.govuk-heading-m', text: 'Defendants')
   end
 
-  def when_viewing_defendant(defendant_nino_or_asn)
-    visit new_laa_reference_path(id: defendant_nino_or_asn)
+  def when_viewing_defendant(_defendant_nino_or_asn)
```

Comment:
this value should be passed as an argument by the looks of it

Reply:
👍 changed to def when_viewing_defendant(defendant_id)